example/src/components/RoomParticipant.tsx

```diff
@@ -0,0 +1,94 @@
+import { BrandColors, AdditionalColors, TextColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import { getShortUsername } from '@utils';
+import React from 'react';
+import { View, StyleSheet, ViewStyle } from 'react-native';
+
+import { Icon } from './Icon';
+import { NoCameraView } from './NoCameraView';
+import { Typo } from './Typo';
+
+type RoomParticipantProps = {
+  participant: Membrane.Participant;
+  pStyle: ViewStyle[];
+};
+
+export const RoomParticipant = ({
+  participant,
+  pStyle,
+}: RoomParticipantProps) => {
+  return (
+    <View style={pStyle}>
+      {!participant.tracks.filter((t) => t.type === 'Video').length ||
+      !participant.tracks.find((t) => t.type === 'Video')!.metadata.active ? (
+        <View style={styles.videoTrack}>
+          <NoCameraView
+            username={getShortUsername(participant.metadata.displayName)}
```

Comment:
wouldn't it be nicer if `NoCameraView` took username and run `getShortUsername`?

Reply:
Done.